Test/SendQueueTest.cs

```diff
@@ -12,13 +12,15 @@ namespace Test
 {
     [TestFixture]
     [NonParallelizable]
+    [Category("SendPacket")]
     public class SendQueueTest
     {
         private const string Filter = "ether proto 0x1234";
         private const int PacketCount = 8;
         private static readonly int DeltaMs = 10;
 
         [Test]
+        [Platform("Win")]
```

Comment:
@chmorgan I added the `[Platform("Win")]` in `SendQueueTest` tests that uses the native API.
this way those tests will only run on windows and would be ignored on any other OS

Reply:
@kayoub5 would you mind if instead we used a IsHardwareAccelerated check instead? Then if the platform had that support it would use the native tests?

Reply:
@kayoub5 and btw I can fix it here, testing azure pipelines now

Reply:
I wanted to avoid relying on `IsHardwareAccelerated` since it could return false if the CI has not been properly configured and winpcap/npcap driver was not properly installed.

Reply:
@kayoub5 really? imo that sounds like a bigger issue if we can't trust that property.

Reply:
@kayoub5 I also removed the thread.Abort() call in Pcap capture loop to get things to work better on linux with dotnet core. I'm not sure what the implications are of doing that. I think we'll see the same issues we see with the win divert driver where if we aren't receiving traffic (and we are so the thread shuts down). Maybe we need to look at closing the adapter to cause the capture to return in the case of no traffic but that may have implications on settings etc.

Reply:
@chmorgan actually I have a test called `TestIsHardwareAccelerated` that checks that `IsHardwareAccelerated` is working as expected.
The moment someone implements send queue for linux or mac (not likely to happen any time soon, since it has to be implemented at driver level first), the test will fail, and we handle it when it happens.
for now the test is
```
            Assert.AreEqual(
                SendQueue.IsHardwareAccelerated,
                Environment.OSVersion.Platform == PlatformID.Win32NT
            );
```

Reply:
@chmorgan I got some free time, if you want to speed things up (rather then discussing with comments), you could setup a gitter room, (takes 2min).
https://gitter.im/home/explore#createcommunity

Reply:
@kayoub5 glitter.im/SharpPcap